### PR TITLE
Error 500

### PR DESCRIPTION
**Pull request by guillim**

### CODE REVIEW

File: packages/twenty-server/src/engine/core-modules/exception-handler/exception-handler.module.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { DynamicModule, Global, Module } from '@nestjs/common';
 
+import { ErrorHandlerService } from 'src/engine/core-modules/auth/auth-exception-handler.service';
```
[the remark this answers is not in the document]
Comment (guillim) [reply packages/twenty-server/src/engine/core-modules/exception-handler/exception-handler.module.ts:3]: After what we discussed, and to be faster, i did not create a dedicatedmodule. If you think it's better and the time for it, I will do !

File: packages/twenty-server/src/engine/core-modules/auth/filters/auth-graphql-api-exception.filter.ts [hunk 2]
```diff
@@ -29,6 +29,7 @@ export class AuthGraphqlApiExceptionFilter implements ExceptionFilter {
       case AuthExceptionCode.INVALID_DATA:
       case AuthExceptionCode.INTERNAL_SERVER_ERROR:
       default:
+        console.log('AuthGraphqlApiExceptionFilter');
```
Comment (charlesBochet) [inline packages/twenty-server/src/engine/core-modules/auth/filters/auth-graphql-api-exception.filter.ts:32]: console.log()

File: packages/twenty-server/src/engine/core-modules/auth/filters/auth-rest-api-exception.filter.ts [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
-  ExceptionFilter,
-  InternalServerErrorException,
-  NotFoundException,
-  UnauthorizedException,
-} from '@nestjs/common';
+import { ArgumentsHost, Catch, ExceptionFilter } from '@nestjs/common';
+
+import { Response } from 'express';
 
+import { ErrorHandlerService } from 'src/engine/core-modules/auth/auth-exception-handler.service';
```
Comment (charlesBochet) [inline packages/twenty-server/src/engine/core-modules/auth/filters/auth-rest-api-exception.filter.ts:5]: service name should match file name AuthExceptionHandlerService

File: packages/twenty-server/src/engine/core-modules/exception-handler/exception-handler.module.ts [hunk 4]
```diff
@@ -13,8 +14,8 @@ import { ExceptionHandlerDriver } from 'src/engine/core-modules/exception-handle
 
 @Global()
 @Module({
-  providers: [ExceptionHandlerService],
-  exports: [ExceptionHandlerService],
+  providers: [ExceptionHandlerService, ErrorHandlerService],
```
Comment (charlesBochet) [inline packages/twenty-server/src/engine/core-modules/exception-handler/exception-handler.module.ts:17]: This services should be exposed in auth module

**Review by charlesBochet — APPROVED** [review]

**Pull request merged**